eth/ethconfig/config.go

```diff
@@ -115,6 +115,7 @@ type Config struct {
 	DirectBroadcast     bool
 	DisableSnapProtocol bool //Whether disable snap protocol
 	EnableTrustProtocol bool //Whether enable trust protocol
+	DisableBscProtocol  bool //Whether disable bsc protocol
```

Comment:
when  define of `Config` changed, gen_config.go should be regenerated

<img width="671" alt="image" src="https://github.com/bnb-chain/bsc/assets/7995985/bec1302c-694f-4641-afb3-1db747598bfd">